packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -94,6 +96,13 @@ const AppModals = ({
         }
     }, [is_logged_in]);
 
+    if (temp_session_signup_params && window.location.href.includes('appstore/onboarding')) {
```

Comment:
```suggestion
    if (temp_session_signup_params && window.location.href === routes.onboarding)) {
```

Reply:
I think we can't do this because routes.onbarding = 'appstore/onbaording' but i think we can do .includes(routes.onboading) instead

Reply:
yeap routes.onboarding should work 😸